public/gdb/treatment.md

```diff
@@ -87,9 +89,9 @@ A newly developing area of bottom surgery is in AMAB non-binary operations which
 
   - Bilateral Mastectomy (breast tissue removal) with chest reconstruction.
 
-**Facial Feminizing / Masculinizing Surgery** (modifications to the skull, cartilage and skin on the face).
+**Facial Feminization / Masculinization Surgery** (modifications to the skull, cartilage, and skin on the face).
 
-  The younger a person is, the less they will need these surgeries, especially if they medically transition prior to the age of 20.
+  The younger a person is, the less they will need these surgeries, especially if they medically transition during puberty.
```

Comment:
This is socially inaccurate. Most people think of puberty ending by the age of 18, which actually the body is still undergoing developmental growth until the age of 25.

I chose the age of 20 as a sensible middle ground, but most importantly the statement "during puberty" applies to a very wide range of years, through which any number of irreversible changes may happen. Before the age of 20, however, most of the severe bone structure changes have not happened yet.

Reply:
That's a fair point.